README.md

```diff
@@ -213,4 +269,27 @@ public class MySlacker {
   // then just use the client!
 }
 ```
-```
+
+## Contributors
+ - [@szabowexler](https://github.com/szabowexler) [:computer:](https://github.com/HubSpot/slack-client/commits?author=szabowexler)
+ - [@zklapow](https://github.com/zklapow) [:computer:](https://github.com/HubSpot/slack-client/commits?author=zklapow)
+ - [@wsorenson](https://github.com/wsorenson) [:computer:](https://github.com/HubSpot/slack-client/commits?author=wsorenson)
```

Comment:
I don't think he's actively working on this atm (based on client history)

Reply:
i was just trying to give prior credit